src/components/appheader.main.jsx

```diff
@@ -104,6 +104,7 @@ class AppHeaderMain extends React.Component {
       isOffline, cartData, isLoading, isSearchFocused,
     } = this.state;
     const isInStandaloneMode = window.navigator.standalone;
+    const isB2B = localStorage.getItem(`${Config.cortexApi.scope}_b2bCart`);
```

Comment:
I suggest renaming this var to something other than `isB2B` for readability. The current name implies it knows if the flow is B2B, but it really only determines if a cart has been selected